FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# unreleased
+- [fixed] Fixes utm parameters not being returned to dynamic link when using universal links
```

Comment:
If this fixes a reported GitHub Issue, please add the issue number in parenthesis after the note. And add the issue number to the PR description - like "Fix #xxxx".